tokio/src/runtime/builder.rs

```diff
@@ -180,7 +180,13 @@ impl Builder {
     ///
     /// # Default
     ///
-    /// The default value is the number of cores available to the system.
+    /// The default value is the number of virtual CPU cores available to the system.
+    ///
+    /// Caution!: This setup may result in degraded performance under certain setups.
+    ///
+    /// The best way to decide on this configuration is through benchmarks.
+    ///
+    /// See the examples below for alternate setups.
```

Comment:
I don't think this needs to be written in such a scary way.

Reply:
@Darksonn thanks for the feedback.

This is just a suggestion to initiate the discussion.

Feel free to propose or edit as you see fit.

Reply:
Sorry, but I've fallen quite behind on reviewing PRs, and I'm going to close this so I can focus on other PRs.